source/hardware/controllers/denon_mc7000.rst

```diff
@@ -23,9 +27,9 @@ Compatibility
    **5.4.21**. All kernel versions **5.6.x** and following are supporting the device.
    All recent Linux distributions are shipped with Audio support for the MC7000, like
 
-  - Ubuntu 20.04 and 20.10
+  - Ubuntu 20.04 & 20.10
```

Comment:
I think we can remove the (incomplete) enumeration of all Linux distributions that are known to work. Basically all do, including those that are not listed (NixOS, Fedora, ...). Instead we should only list those that are known **not** to work if there are any. But I don't think this is needed.

Reply:
You maybe right - that was a requirement a year ago when the ALSA patch just arrived and almost non of the Distributions delivered a Kernel that supported the sound device. I would still keep the information in for the Kernel version starting to support the MC7000

Reply:
Yes, mentioning the kernel version is helpful :+1: